x-pack/legacy/plugins/snapshot_restore/public/app/components/show_hide_indices.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { EuiDescriptionListDescription, EuiTitle, EuiLink, EuiIcon } from '@elastic/eui';
+interface Props {
+  indices: string[] | string | undefined;
+  defaultState: boolean;
+  i18nId: string;
+}
+
+import { useAppDependencies } from '../index';
+
+export const ShowHideIndices: React.FunctionComponent<Props> = ({
+  indices,
+  defaultState,
+  i18nId,
+}) => {
+  const {
+    core: { i18n },
+  } = useAppDependencies();
+  const { FormattedMessage } = i18n;
+  const [isShowingFullIndicesList, setIsShowingFullIndicesList] = useState<boolean>(defaultState);
```

Comment:
Looks like the `defaultState` is always false. Does it need to be passed as a `prop`?

Reply:
Fixed. Removed.